tests/e2e/specs/price-benchmark/price-benchmark-suggestions.test.js

```diff
@@ -132,30 +182,7 @@ test.describe( 'Price Benchmark Page', () => {
 		test( 'Shows empty state notice when Market Insights is not enabled for the account.', async () => {
 			await priceBenchmarkPage.fulfillPriceBenchmarkSuggestions(
 				{
-					message: {
-						error: {
-							code: 403,
-							message:
-								'Market Insights not enabled for account 5330359695. For more information check https://support.google.com/merchants/answer/9625913',
-							errors: [
-								{
-									message:
-										'Market Insights not enabled for account 5330359695. For more information check https://support.google.com/merchants/answer/9625913',
-									domain: 'global',
-									reason: 'forbidden',
-								},
-							],
-							status: 'PERMISSION_DENIED',
-							details: [
-								{
-									'@type':
-										'type.googleapis.com/google.rpc.ErrorInfo',
-									reason: 'forbidden',
-									domain: 'global',
-								},
-							],
-						},
-					},
+					message: JSON.stringify( ACCOUNT_NOT_ENABLED_ERROR ),
```

Comment:
@ankitrox There was an issue with how the payload causing the test to give a false positive. Prior to that change, the snackbar would still show up when testing. This is because invalid JSON was being sent from the mocked response, causing the JSON.parse [line](https://github.com/woocommerce/google-listings-and-ads/blob/feature/2824-price-benchmarks/js/src/data/actions.js#L1247) to fail, thus the [following lines](https://github.com/woocommerce/google-listings-and-ads/blob/feature/2824-price-benchmarks/js/src/data/actions.js#L1250-L1254) were never executed. I've pushed a fix and should be fine now. cc @mukeshpanchal27